File: portal/portal/App_Code/Categoria.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace portal.App_Code
{
    public class Categoria
    {
        private int _codigo;
        private string _nomeTipo;
        private string _descritivo;

        public int Codigo
        {
            get
            {
                return _codigo;
            }

            set
            {
                _codigo = value;
            }
        }

        public string NomeTipo
        {
            get
            {
                return _nomeTipo;
            }

            set
            {
                _nomeTipo = value;
            }
        }

        public string Descritivo
        {
            get
            {
                return _descritivo;
            }

            set
            {
                _descritivo = value;
            }
        }
    }
}

File: portal/portal/App_Code/Categoria.php
<?php
####################################################################################################################
/* CATEGORIA: Classifica tipos de conteúdo competente de cada disciplina, "grupos de conteúdo". -> A Página é personalizada para aquele tipo de conteúdo. */
####################################################################################################################

namespace portal\App_Code {
	class Categoria {
		//ATRIBUTES----------------------------------------------------------------------
			//MySQL <~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~> IO
			private $codigo;
			private $nomeTipo;
			private $descritivo;
		//CONSTRUTOR---------------------------------------------------------------------
			public function __construct(){}
        
            public function  setCategoria ($cod,$nomTip,$desc){
				$this -> codigo = $cod;
                $this -> nomeTipo = $nomTip;
				$this -> descritivo= $desc;
			}
		//SETS---------------------------------------------------------------------------
			public function setCodigo($cod){
				$this -> codigo = $cod;
			}
			public function setNomeTipo($nomTip){
				$this -> nomeTipo = $nomTip;
			}			
			public function setDescritivo($desc){
				$this -> descritivo= $desc;
			}
		//GETS---------------------------------------------------------------------------
			public function getCodigo(){
				return $this -> codigo;
			}
			public function getNomeTipo(){
				return $this -> nomeTipo;
			}
			public function getDescritivo(){
				return $this -> descritivo;
			}
	}
}
?>
